marge/app.py

```diff
@@ -11,13 +11,14 @@
 import tempfile
 from datetime import timedelta
 
+import configargparse
```

Comment:
@mrusu91 You are still importing `argparse` and it's being used (e.g. raising `argparse.ArgumentTypeError`). I'd prefer if we use only `configargparse` everywhere. Maybe we can just do:
```python
import configargparse as argparse
```

Reply:
will do that, but maybe keep using the original name `configargparse` because I think it keeps  things clear?

Reply:
:+1: